### DISCUSSION TITLE
Audio and image not loading

### DISCUSSION THREAD

**Discussion by gametamilzhastudio** [microfeed/microfeed#118] [Q&A]
Hi, 
I am adding audio/ video it gets the size of the content. but, it is not loading the content. Due to this episode is not fetching in Spotify or apple podcasts. can anyone help me :)
I have attached following problem.

![Screenshot 2024-01-22 151041](https://github.com/microfeed/microfeed/assets/49088939/432427ca-a42f-401c-be86-b3da85e0ceb1)
![Screenshot 2024-01-22 151012](https://github.com/microfeed/microfeed/assets/49088939/484c0452-0283-4a9c-829f-25758cdb35ce)
![Screenshot 2024-01-22 150939](https://github.com/microfeed/microfeed/assets/49088939/8b3a123a-edd7-4151-9245-8b2a8486c15b)

**Comment by wenbinf** [comment]
Hi @gametamilzhastudio 

The root cause is that you don't setup R2 public url correctly. As you can see, the base url of the uploaded file is https://podcast.biggmic.com/ in your screenshot, which is incorrect.

The fix is easy - 

Please follow the instructions on the home page of your /admin dashboard:
https://podcast.biggmic.com/admin

<img width="831" alt="Screenshot 2024-01-22 at 10 11 30 AM" src="https://github.com/microfeed/microfeed/assets/1719237/9f507640-77fe-430e-96f5-fa2536bc1589">

**Reply by gametamilzhastudio** [reply]
Hi fang, Big fan of your products. Glad to connect with you. 

I have another question how to rename our rss feed ?

**Reply by gametamilzhastudio** [reply]
and i couldn't upload the files

![Screenshot 2024-01-23 152455](https://github.com/microfeed/microfeed/assets/49088939/7ed46c55-8516-41ee-9848-7a0a17843e29)

**Comment by wenbinf** [comment]
> Hi fang, Big fan of your products. Glad to connect with you.
> 
> I have another question how to rename our rss feed ?

What do you mean by "renaming rss feed"? Do you want to use a different path name in the url?

**Comment by wenbinf** [comment]
> and i couldn't upload the files
> 
> ![Screenshot 2024-01-23 152455](https://private-user-images.githubusercontent.com/49088939/298890130-7ed46c55-8516-41ee-9848-7a0a17843e29.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYwMzUwMTAsIm5iZiI6MTcwNjAzNDcxMCwicGF0aCI6Ii80OTA4ODkzOS8yOTg4OTAxMzAtN2VkNDZjNTUtODUxNi00MWVlLTk4NDgtN2EwYTE3ODQzZTI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIzVDE4MzE1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYxOGYyOTg5MmZmOWU0ZDdhNGQ5MzIxMWU1ZDU5OTE4MTk0N2ZhYzMzODMzYjA4OWVkZWVkY2MwNzY1YWQxOTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hoqhhlQdZdCdVSxJWAJUcy9d9ZTcrpHdZhj4k0ypfKc)

Please double check if your R2 api key is correct: https://github.com/microfeed/microfeed?tab=readme-ov-file#step-2-put-some-secrets-on-your-forked-repo

**Reply by gametamilzhastudio** [reply]
Thanks Fang,
It works :)

regarding collaborations how should I contact you?
looking forward to hearing from you:)

**Reply by wenbinf** [reply]
awesome! glad it works for you.

please feel free to contact us at [support@microfeed.org](mailto:support@microfeed.org)

**Reply by gametamilzhastudio** [reply]
I have already sent a mail at support@microfeed.org fang!

**Reply by gametamilzhastudio** [reply]
Hi @wenbinf can we use the Amazon web server or any other web server?

**Reply by wenbinf** [reply]
No. microfeed is serverless with cloudflare only.

**Reply by gametamilzhastudio** [reply]
Okies. Thanks @wenbinf  :) i wanted to connect with you in person, let me know how we can:)
